script/build-icons

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
```

Comment:
I guess we should probably set some sane defaults:

```suggestion
#!/usr/bin/env bash

set -o errexit
set -o pipefail
set -o nounset

```

— https://kvz.io/bash-best-practices.html

Reply:
Yes! Setting these defaults is definitely 💯 

We don't necessarily have to move from `sh` to `bash` though in my opinion 🤷 . We're not really doing anything too fancy here.

Reply:
> We don't necessarily have to move from `sh` to `bash` though in my opinion 🤷 . We're not really doing anything too fancy here.

Totally cool with me. Just wasn't 100% sure - from the top of my head - whether `sh` supports these options. ✌️

Reply:
I was actually surprised as well 🤓. But according to `shellcheck` it supports all but the one executing on failures within pipes (which is fine as we don't use it).